Octokit/Models/Request/Enterprise/UpdatePreReceiveEnvironment.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Describes an update to an existing pre-receive environment.
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class UpdatePreReceiveEnvironment
```

Comment:
do you know if the `Name` or `ImageUrl` are required parameters for the update call?  Wondering if there should be a constructor that takes these

Reply:
Both are optional, and I have integration tests for omitting one or the other.

The call also works when no parameters are passed, although that doesn't really serve a purpose.
So, we could
1. Add a unit tests for the update nothing scenario
2. Create a constructor that ensures that at least one value is set. Can't really do separate constructors as both values are strings.
3. Leave it as is